src/scss/layout/_video.scss

```diff
@@ -0,0 +1,57 @@
+@mixin  fontStyle_video ($color, $fontSize, $fontWeight) {
+    color: $color;
+    font-size: $fontSize;
+    font-weight: $fontWeight;
+    line-height: 28px;
+    font-family: Helvetica;
+    font-style: normal;
+}
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0px;
+}
+
+body {
+    max-width: 1440px;
+    margin: auto;
+}
+
+.container {
+    margin: 80px 80px 88px 83px;
+}
+
+.info-video__section {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: auto;
+}
+
+.paragraph {
+    grid-column: 1/2;
+    grid-row: 1/2;
+    align-self: center;
+    max-width: 626px;
+}
+
+.paragraph_title p {
+    @include fontStyle_video(#577598, 24px, bold);
+    margin-bottom: 20px;
+}
+
+.paragraph_text {
+    @include fontStyle_video(#8894B4, 24px, normal);
+    margin: 0px;
+    max-width: 518px;
+}
+
+.video-section {
+    margin-left: 25px;
+    max-width: 628px;
+}
+
+video {
```

Comment:
don't use tags as selectors since you could affect other video elements in other components in the page. Please create a new class for it.

Reply:
Done.

Reply:
it is still pending